versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx

```diff
@@ -104,15 +103,15 @@ Use the following command to register a new worker cluster with the KubeSlice Co
 kubeslice-cli install --config=<new-worker-topology-yaml> -s controller
 ```
 
-## Onboard Namespaces
+## Create a Slice
 
 To onboard your existing namespaces (and their applications) onto a slice, follow these steps:
```

Comment:
onto a slice: 

follow these steps is redundant

Reply:
done